dev/ci/ci-common.sh

```diff
@@ -15,18 +15,29 @@ then
     then
         export CI_PULL_REQUEST="${CI_BRANCH#pr-}"
     fi
+# Travis build in -local mode
+elif [ -n "${TRAVIS}" ];
```

Comment:
Do you mind putting the comments after the condition they apply to? Currently it feels like they have half a foot in the previous block.

Reply:
Sure, I never know where to place them, I usually prefer to read first the comment then the condition but will update.